homeassistant/components/monzo/const.py

```diff
@@ -0,0 +1,16 @@
+"""Constants for the Monzo integration."""
+
+DOMAIN = "monzo"
+
+OAUTH2_AUTHORIZE = "https://auth.monzo.com"
+OAUTH2_TOKEN = "https://api.monzo.com/oauth2/token"
```

Comment:
These only seem to be used in application_credentials.py, if that's the case, move them there.